logger/logger.go

```diff
@@ -34,11 +34,15 @@ func (l *Logger) newLogEvent(data, header string, w io.Writer) error {
 	if data == "" || data[len(data)-1] != '\n' {
 		e.data = append(e.data, '\n')
 	}
-	e.output.Write(e.data)
+	_, err := e.output.Write(e.data)
+	if err != nil {
+		fmt.Printf("Log Write failed: %v\n", err)
```

Comment:
We return an err if io.Writer is nil but we currently don't act on it in any of the calling funcs, but report on this one (as well as returning it). Could simplify this by reporting both issues in the calling func if err != nil

Reply:
Okay made the change

I pass the error back down from newLogEvent to each sub logger and handle the display in there
there is not much that can be done so at the moment all it does is display the error message

I was unsure on fmt v log to display the message so i went with standard logger to at least include timestamp etc on the chance that the gct logger fails to write